### PR TITLE
Ta 3 auto service classes and objects task

### PR DESCRIPTION
**Pull request by dimatana**
test1

### CODE REVIEW

File: .idea/uiDesigner.xml
Comment (jackcucu) [inline .idea/uiDesigner.xml:1]: read a about .gitignore file after that add file and ignore all .idea

File: src/main/java/dima/tanasciuc/Main.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package md.tekwillacademy;
+package dima.tanasciuc;
```
Comment (jackcucu) [inline src/main/java/dima/tanasciuc/Main.java]: md.dima.tanasciuc 
https://www.baeldung.com/java-packages#1-naming-conventions

File: .idea/uiDesigner.xml
Comment (dimatana) [reply .idea/uiDesigner.xml:1]: do you want me to understand that in 6 days?

Comment (jackcucu) [reply .idea/uiDesigner.xml:1]: Hule,daca faci bine deodată o sa faci bine mereu 😅

**Review by jackcucu — APPROVED** [review]

**Pull request merged**